test/new-e2e/system-probe/config/platforms.json

```diff
@@ -0,0 +1,36 @@
+{
+    "url_base": "https://dd-agent-omnibus.s3.amazonaws.com/kernel-version-testing/rootfs",
+    "x86_64": {
+        "ubuntu_16.04": "master/xenial-server-cloudimg-x86_64.qcow2.xz",
+        "ubuntu_18.04": "master/bionic-server-cloudimg-x86_64.qcow2.xz",
+        "ubuntu_20.04": "master/focal-server-cloudimg-x86_64.qcow2.xz",
+        "ubuntu_22.04": "master/jammy-server-cloudimg-x86_64.qcow2.xz",
+        "ubuntu_23.10": "master/mantic-server-cloudimg-x86_64.qcow2.xz",
+        "amzn_4.14": "master/amzn2-kvm-2.0-x86_64-4.14.qcow2.xz",
+        "amzn_5.4": "master/amzn2-kvm-2.0-x86_64-5.4.qcow2.xz",
+        "amzn_5.10": "master/amzn2-kvm-2.0-x86_64-5.10.qcow2.xz",
+        "fedora_37": "master/Fedora-Cloud-Base-37.x86_64.qcow2.xz",
+        "fedora_38": "master/Fedora-Cloud-Base-38.x86_64.qcow2.xz",
+        "debian_10": "master/debian-10-generic-x86_64.qcow2.xz",
+        "debian_11": "master/debian-11-generic-x86_64.qcow2.xz",
+        "debian_12": "master/debian-12-generic-x86_64.qcow2.xz",
+        "centos_79": "master/centos-7.9-x86_64.qcow2.xz",
+        "centos_8": "master/centos-8-x86_64.qcow2.xz"
+    },
+    "arm64": {
+        "ubuntu_18.04": "master/bionic-server-cloudimg-arm64.qcow2.xz",
+        "ubuntu_20.04": "master/focal-server-cloudimg-arm64.qcow2.xz",
+        "ubuntu_22.04": "master/jammy-server-cloudimg-arm64.qcow2.xz",
+        "ubuntu_23.10": "master/mantic-server-cloudimg-arm64.qcow2.xz",
```

Comment:
these should be updated to use the new path: master/ubuntu-<version_num>-arm64.qcow2.xz